docs/USERMODEL.md

```diff
@@ -0,0 +1,74 @@
+# Description of the user model of Entirety
+
+Entirety comes with two different ways to assign permissions to users: Roles and Groups.
+Rules come with a set of permissions that are application-wide while groups grand project-wide permissions.
```

Comment:
Do you mean roles?

Reply:
Where exactly? 
I only found a typo: grand -> grant.

Reply:
In line 3 (and the the other parts of the doc) there are Roles and Groups. Line 4 describes rules.

Reply:
Ah, thanks :) got it.